### DISCUSSION TITLE
API docs

### DISCUSSION THREAD

**Discussion by astrakid** [OpenSignLabs/OpenSign#1338] [General]
hi,
i am looking for the API documentation. how can i use APIs?
i set up opensign successully (as some kind of PoC for my needs) and have to test how to integrate it with the existting CRM (closed software, so i have to check how to get the information transfered to opensign).

regards,
andre

**Comment by andrew-opensignlabs** [comment]
APIs are not available on the free self-hosted version. Please contact hello@opensignlabs.com if you want to use APIs with your self-hosted version. You can find the API docs [here](https://docs.opensignlabs.com).

**Reply by astrakid** [reply]
omg - i overlooked the navigation bar on the left. thx.
